pyproject.toml

```diff
@@ -8,7 +8,7 @@ license = "MIT"
 [tool.poetry.dependencies]
 python = "^3.11"
 homeassistant = "^2023.11"
-mytoyota = "2.1.1"
+mytoyota = "2.1.4"

```

Comment:
We should use a range to allow all non major releases of the mytoyota lib. This will give us more freedom for upcoming releases. Please have a look at my Pull Requests here: https://github.com/DurgNomis-drol/ha_toyota/pull/257